testproject/tests/test_backends.py

```diff
@@ -78,3 +79,13 @@ def test_yubikey_backend(active_user_with_many_otp_methods, settings):
     auth_method = user.mfa_methods.get(name="yubi")
     dispatcher = YubiKeyMessageDispatcher(mfa_method=auth_method, config=config)
     dispatcher.confirm_activation(code)
+
+
+@pytest.mark.django_db
+def test_sms_aws_backend_without_credentials(active_user_with_sms_aws_otp, settings):
+    auth_method = active_user_with_sms_aws_otp.mfa_methods.get(name="sms_aws")
+    conf = settings.TRENCH_AUTH["MFA_METHODS"]["sms_aws"]
```

Comment:
If we configure the sms_aws setting in our app, tests will fail because response data details will give us another message. To get always the same error message we should add this line:
conf["AWS_REGION"] = "region"